tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -1212,6 +1243,27 @@ static uint8_t set_filter_list(const void *cmd, uint16_t cmd_len,
 	return BTP_STATUS_SUCCESS;
 }
 
+static uint8_t set_extended_advertising(const void *cmd, uint16_t cmd_len,
+					void *rsp, uint16_t *rsp_len)
+{
+	const struct btp_gap_set_extended_advertising_cmd *cp = cmd;
+	struct btp_gap_set_extended_advertising_rp *rp = rsp;
+
+	LOG_DBG("ext adv settings: %d", cp->settings);
```

Comment:
```suggestion
	LOG_DBG("ext adv settings: %u", cp->settings);
```

`cp->settings` is an unsigned value